app/src/styles.scss

```diff
@@ -11,3 +11,48 @@ html {
 body {
   margin: 0;
 }
+
+.adf-pagination__block {
```

Comment:
can you please extract all the pagination overrides into the separate file, or here: https://github.com/Alfresco/alfresco-content-app/blob/develop/app/src/app/ui/overrides/adf-style-fixes.theme.scss#L1